torch/_subclasses/fake_tensor.py

```diff
@@ -784,9 +796,15 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
             return converter(self, args[0])
 
         if self.check_for_non_fake(flat_arg_tensors):
-            raise Exception(
-                "Invoking operators with non-Fake Tensor inputs in FakeTensorMode is not yet supported. "
-                f"Please convert all Tensors to FakeTensors first. Found in {func}(*{args}, **{kwargs})"
+            if not CONSTANT_LIFTING:
+                raise Exception(
+                    "Invoking operators with non-Fake Tensor inputs in FakeTensorMode is not yet supported. "
+                    f"Please convert all Tensors to FakeTensors first. Found in {func}(*{args}, **{kwargs})"
+                )
+            args, kwargs = tree_map_only(
```

Comment:
as @ezyang commented, I think we need to keep track of the inputs we lift, and throw if we call torch.Tag.inplace_view` on them.

```
x = torch.rand([4, 4])
with FakeTensorMode(allow_non_fake_inputs=True):
     y = x.add_(3)
     y.resize_([16])
     assert(x.shape == y.shape) # would fail
```